### DISCUSSION TITLE
Add docs content

### DISCUSSION THREAD

**Discussion by OxNinja** [thuliteio/doks#477] [Q&A] [answered]
Hello, I am messing around with this beautiful theme, and want to add some docs

I want to create a new sub folder like so:
![image](https://user-images.githubusercontent.com/57400445/134890506-52fbec0d-62ba-4093-b7e4-7ed8c7c47354.png)

And delete the default `prologue` and `help`, unfortunately I can't find in the documentation at getdoks.org or in the source code how to do so?

**Comment by h-enk** [comment]
See: https://getdoks.org/tutorial/add-menus/#add-to-docs-menu

If you're using the child theme: https://github.com/h-enk/doks-child-theme/blob/master/config/_default/config.toml#L98-L100

In short: comment out (or delete) what you don't need.

**Reply by h-enk** [reply]
Note: auto generated menus will be a feature for the (nearby) future

**Reply by OxNinja** [reply]
Yes I already read that and it does not work as intended:

I create a sub dir `test` with MD files inside, the TEST dir appears in the docs but not the files inside,
I delete the prologue dir but it persists on the website.

https://github.com/h-enk/doks-child-theme/blob/master/config/_default/config.toml#L98-L100

What should I do with that? It is already commented in my config

If wanted I can give source code

Thanks for your time

**Reply by h-enk** [reply]
> If wanted I can give source code

Yes, that would make it easier to help you out

**Reply by OxNinja** [reply]
https://git.fakenews.sh/ctfkit/ctf-kit-website/-/tree/idea/02-doks
See `/screen.png` for a visual hint.

Thanks again

**Comment by h-enk** [comment, marked as answer]
Make sure that:
1. you are referencing the correct parent identifier (set in `menus.toml`) as the parent menu item — e.g. `ctfkit`, and
2. the page title is unique — make it e.g. `CTF Kit intro`

E.g. for `./content/docs/ctfkit/introduction.md`:

```md
---
title: "CTF Kit intro"
description: "Doks is a Hugo theme helping you build modern documentation websites that are secure, fast, and SEO-ready — by default."
lead: "Doks is a Hugo theme helping you build modern documentation websites that are secure, fast, and SEO-ready — by default."
date: 2020-10-06T08:48:57+00:00
lastmod: 2020-10-06T08:48:57+00:00
draft: false
images: []
menu:
  docs:
    parent: "ctfkit"
weight: 100
toc: true
---
```

Then you will get:

![Snag_2ee8fad](https://user-images.githubusercontent.com/3902872/135217312-43cc7f40-38fb-45d6-bb56-6b837e6b7834.png)

**Comment by h-enk** [comment]
When you delete pages, also make sure to delete references (in pages + menus) to the deleted pages.

E.g. `[Quick Start →](/)` + `[Help →]({{< relref "how-to-update" >}})` in `./content/docs/ctfkit/introduction.md`.

After deleting prologue + help dirs + deleting references:

![Snag_2f5f981](https://user-images.githubusercontent.com/3902872/135218391-2d4ba464-f9f0-4ac7-9095-3c15e7f45010.png)
